src/main/resources/assets/immersiveengineering/manual/en_us/storage_minecarts.txt

```diff
@@ -0,0 +1,6 @@
+Storage Minecarts
+Let The Bullgine Run
```

Comment:
Can't help but notice that you overrode me in the end ;)

Reply:
Yeah, I liked this shanty better. I can change it to the one you suggested if you want?